File: src/main/java/org/jcodec/containers/dpx/TelevisionHeader.java
package org.jcodec.containers.dpx;

public class TelevisionHeader {
    public int timecode;
    public int userBits;
    public byte interlace; // Interlace (0 = noninterlaced; 1 = 2:1 interlace)
    public byte filedNumber;
    public byte videoSignalStarted;
    public byte zero;
    public int horSamplingRateHz;
    public int vertSampleRateHz;
    public int frameRate;
    public int timeOffset;
    public int gamma;
    public int blackLevel;
    public int blackGain;
    public int breakpoint;
    public int referenceWhiteLevel;
    public int integrationTime;
//        public byte[] reserved;

}

File: src/main/java/org/jcodec/containers/dpx/TelevisionHeader.kt
package org.jcodec.containers.dpx

class TelevisionHeader {
    var timecode = 0
    var userBits = 0
    var interlace // Interlace (0 = noninterlaced; 1 = 2:1 interlace)
            : Byte = 0
    var filedNumber: Byte = 0
    var videoSignalStarted: Byte = 0
    var zero: Byte = 0
    var horSamplingRateHz = 0
    var vertSampleRateHz = 0
    var frameRate = 0
    var timeOffset = 0
    var gamma = 0
    var blackLevel = 0
    var blackGain = 0
    var breakpoint = 0
    var referenceWhiteLevel = 0
    var integrationTime = 0 //        public byte[] reserved;
}